third_party/utf8proc/BUILD.bazel

```diff
@@ -0,0 +1,14 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "utf8proc",
+    srcs = glob([
+        "*.cpp",
+    ]),
+    hdrs = glob([
+        "include/*.h",
+    ]),
+    visibility = ["//visibility:public"],
+    deps = [],
```

Comment:
If this library doesn't have any deps, you don't need to write deps = [].

Reply:
removed